modules/storages/app/common/storages/peripherals/registry.rb

```diff
@@ -28,13 +28,16 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
+require "dry/auto_inject"
+
 module Storages
   module Peripherals
     class Registry
       extend Dry::Container::Mixin
 
       class Resolver < Dry::Container::Resolver
         def call(container, key)
+          Rails.logger.tagged("Registry").info("Resolving #{key}")
```

Comment:
There is another one hiding here :face_with_peeking_eye:

Reply:
Damn. >.< That's what I get for working on the same code for too long. xD